scripts/prepare_bootstrap_inputs.py

```
@@ -0,0 +1,294 @@
+- masks sensitive SSH keys in logs.
+"""
+
+from __future__ import annotations
+
+import base64
+import json
+import os
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Callable
```

Ruff (0.14.6)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)